recipes-ni/nisdbootconfig/files/nisdbootconfig

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# check if target is NOT using ubifs (mtd)
+if ! cat /proc/mtd | grep -qs mtd ; then
+
+    # configure the partitions to mount by label in fstab
+    sed -i '/ubi/d' /etc/fstab
+    echo LABEL=nibootfs         /boot                ext4       sync           0  0 >>/etc/fstab
+    echo LABEL=niconfig         /etc/natinst/share   ext4       sync           0  0 >>/etc/fstab
+    mkdir -p /etc/natinst/share
+
+    # configure the path for fw_printenv to read environmental variables from u-boot
+    sed -i '/mtd/d' /etc/fw_env.config
```

Comment:
For my understanding, could you provide examples of what lines you expect to be deleted by this command?

Reply:
Here's the default values for fw_env.config:
/dev/mtd4		0x0000		0x20000		0x20000
/dev/mtd5		0x0000		0x20000		0x20000

I'll add more description in the comments to note this down.